### ISSUE TITLE
Opening terminals in lf

### ISSUE THREAD

**Issue by DomBito** [LukeSmithxyz/voidrice#1002]
Imgine the workflow:

1. You are navigating with `lf`.
2. You need a terminal to work on that directory _while keeping `lf` active_.
3. In `lf`'s shell you type `st &` because that the fastest option.
4. You continue using `lf` while working on that terminal.
5. Suddenly you don't need that lf instance anymore and forget the terminal one depends on it.
6. You quit `lf` and the terminal gets killed.

So putting this in lfrc should make thing easier:
```
cmd openterminal $nohup $TERMINAL >/dev/null &
map \\ openterminal
#map W openterminal #alternative
```
Escape is good because it's nearby the shell key, which is Enter. W is also nice, since w also opens a terminal that covers your `lf` window.

An even better option would be to have a toggle key to either open things normaly or disowned by nohup in a separated window, but I don't know how to do that.

**Comment by TheYellowArchitect** [comment]
lol what a coincidence, I was trying to find a solution to this problem as well.

tl;dr: `nohup` is the command which helps in this.

For your example `nohup st &` works, and they are independent, you can kill one, and the other doesn't die

**Comment by LukeSmithxyz** [comment]
Added. I will say that you might find that `setsid -f` is a bit more robust and clean than `nohup` though.
